src/Pages/HumansAtNasa/HumansAtNasa.tsx

```diff
@@ -0,0 +1,18 @@
+﻿﻿import React from "react";
+import {NavBar} from "../../NavBar/NavBar";
+import {HumanAtNasa} from "../../Components/HumanAtNasa/HumanAtNasa";
+import './HumansAtNasa.scss';
+
+export function HumansAtNasa() {
+    return (
+        <div className='humansAtNasaBody'>
```

Comment:
Could we make this a `section` instead of a `div`?

Reply:
its a section now.